controllers/velero_test.go

```diff
@@ -31,7 +31,7 @@ const (
 	proxyEnvValue                          = "http://proxy.example.com:8080"
 	argsMetricsPortTest              int32 = 69420
 	defaultFileSystemBackupTimeout         = "--restic-timeout=1h"
-	defaultRestoreResourcePriorities       = "--restore-resource-priorities=securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotbackups.datamover.oadp.openshift.io,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io"
```

Comment:
as we talked, `datauploads.velero.io` is just for OADP 1.3/Velero 1.12, right?

Reply:
@mateusoliveira43 That's right.